platform/windows/os_windows.cpp

```diff
@@ -2586,7 +2586,15 @@ Error OS_Windows::execute(const String &p_path, const List<String> &p_arguments,
 	modstr.resize(cmdline.size());
 	for (int i = 0; i < cmdline.size(); i++)
 		modstr.write[i] = cmdline[i];
-	int ret = CreateProcessW(NULL, modstr.ptrw(), NULL, NULL, 0, NORMAL_PRIORITY_CLASS, NULL, NULL, si_w, &pi.pi);
+		
```

Comment:
Travis is complaining about trailing space here.

Reply:
I don't get it. What's up with Travis? The trailing space is removed but he's still complaining.

Reply:
The build log shows you what the problem is: https://travis-ci.org/godotengine/godot/jobs/554133774

Your includes are not sorted alphabetically.

Reply:
Also the log doesn't make any sense to me

```
--- a/core/bind/core_bind.cpp	2019-07-04 08:22:46.069083288 +0000
+++ b/core/bind/core_bind.cpp	2019-07-04 08:22:53.631427484 +0000
@@ -36,10 +36,10 @@
 #include "core/io/marshalls.h"
 #include "core/math/crypto_core.h"
 #include "core/math/geometry.h"
+#include "core/method_bind_ext.gen.inc"
 #include "core/os/keyboard.h"
 #include "core/os/os.h"
 #include "core/project_settings.h"
-#include "core/method_bind_ext.gen.inc"
```